libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -172,6 +193,29 @@ export class PickingComponent extends PickingListMixin(LitElement) {
           <h3 class="title-empty">${i18n(`picking.great-job`)}!</h3>
           <p>${i18n(`picking.all-items-are-processed`)}!</p>
         </div>
+        <div class="submit-wrapper">
```

Comment:
Yup, in this case we need it, because we are styling the parent container of the button to apply different BG color and shadow.
![Screenshot 2023-03-29 at 22 32 21](https://user-images.githubusercontent.com/105730125/228672772-6c76e75f-6997-41c9-a8c5-385d63d2b0a8.png)